associations/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rental << rental
```

Comment:
- [ ] you are using both `@rental` and `@rentals` please make sure that after calling it `@rentals` to use this naming convention everywhere.

Reply:
well resolved! ✔️